frontend/src/App.js

```diff
@@ -129,7 +129,7 @@ function App() {
     const colorIdx = color;
     const colorHex = `#${colors[colorIdx]}FF`;
     context.fillStyle = colorHex;
-    context.fillRect(x, y, 1, 1);
+    context.fillRect(x, y, 10, 10);
```

Comment:
This should still be (1, 1), why was it changed?

Reply:
My bad. I pushed test data